zulip/integrations/rss/rss-bot

```diff
@@ -51,6 +51,13 @@ stream every 5 minutes is:
 parser = zulip.add_default_arguments(
     argparse.ArgumentParser(usage)
 )  # type: argparse.ArgumentParser
+parser.add_argument(
+    "--topic",
+    dest="topic",
+    help="The Stream Topic to which to send RSS messages.",
+    default="stream events",
```

Comment:
This changes the default behavior which currently exists; is there a reason for this?

Reply:
I've removed the default value via: https://github.com/zulip/python-zulip-api/pull/747/commits/4de095269de917feb2409c54bd5171bcbcd8f79c

Is my assumption correct that it will now fallback to `data.feed.title or feed_url` if no --topic arg is specified?